usagov/vote-gov-drupal
# VOTE-84: Add additional fields to state/territory content type.

## Jira ticket

[VOTE-84](https://cm-jira.usa.gov/browse/VOTE-84)

## Description

Add additional fields to State/Territory content type.

## Deployment and testing

### Pre-deploy

none

### Post-deploy

1. Run `lando drush cim` locally to import the content type changes.

### QA/Test

1. Login as an admin.
2. Create or edit a State/Territory content item.
3. Confirm that the new fields are available per the requirements document:
https://docs.google.com/spreadsheets/d/1Z8j-a9KLrDlyG4kGqXmfwENlidtXTLYE/edit#gid=1000964088
4. Confirm that the fields which are marked as translatable are available when editing a translated content item.

## Checklist for the Developer




## Checklist for the Peer Reviewers
